.github/workflows/spdx_license.yaml

```diff
@@ -51,3 +51,40 @@ jobs:
         uses: VeryGoodOpenSource/very_good_coverage@v2.1.0
         with:
           path: bricks/spdx_license/hooks/coverage/lcov.info
+
+  build_brick:
+    defaults:
+      run:
+        working-directory: bricks/spdx_license
```

Comment:
nit:

```suggestion
        working-directory: bricks/spdx_license
        
```

Reply:
I will also add this to the `build_hooks` job.